### ISSUE TITLE
log filled up with error Error in mail

### ISSUE THREAD

**Issue by Toasterme** [vijaygill/set-bluetooth-clocks#2]
So i have this error that log a lot of the same thing over and over : 

`
{"log":"\u001b[31m2023-12-01 07:42:01,493 - Error in mail: [org.freedesktop.DBus.Error.ServiceUnknown] The name org.bluez was not provided by any .service files\u001b[0m\n","stream":"stderr","time":"2023-12-01T12:42:01.493840628Z"}
{"log":"\u001b[37m2023-12-01 07:42:01,493 - Main loop started.\u001b[0m\n","stream":"stderr","time":"2023-12-01T12:42:01.493979216Z"}
{"log":"\u001b[31m2023-12-01 07:42:01,495 - Error in mail: [org.freedesktop.DBus.Error.ServiceUnknown] The name org.bluez was not provided by any .service files\u001b[0m\n","stream":"stderr","time":"2023-12-01T12:42:01.495993035Z"}
{"log":"\u001b[37m2023-12-01 07:42:01,496 - Main loop started.\u001b[0m\n","stream":"stderr","time":"2023-12-01T12:42:01.496124154Z"}
{"log":"\u001b[31m2023-12-01 07:42:01,498 - Error in mail: [org.freedesktop.DBus.Error.ServiceUnknown] The name org.bluez was not provided by any .service files\u001b[0m\n","stream":"stderr","time":"2023-12-01T12:42:01.498534127Z"}
`

And it is pretty big 
`
]# ls -al
total 22212280
drwx--x---  4 root root        4096 Dec  1 07:42 .
drwx--x--- 15 root root        4096 Nov  8 15:09 ..
-rw-r-----  1 root root 22745180274 Dec  1 07:42 6aa547ee43f40c9b1610ff7f43b6fd4b461cd84e1bfd5f6c0d7a26f33f13b524-json.log
drwx------  2 root root        4096 Nov  6 17:06 checkpoints
-rw-------  1 root root        4884 Dec  1 07:42 config.v2.json
-rw-------  1 root root        1654 Dec  1 07:42 hostconfig.json
-rw-r--r--  1 root root          13 Nov 13 18:51 hostname
-rw-r--r--  1 root root         150 Dec  1 07:42 hosts
drwx--x---  3 root root        4096 Nov  6 17:06 mounts
-rw-r--r--  1 root root          38 Nov 13 18:51 resolv.conf
-rw-r--r--  1 root root          71 Nov 13 18:51 resolv.conf.hash
`

**Comment by vijaygill** [comment]
@Toasterme - Are you running it in docker or natively on host? Do you have bluez installed in host environment? 

![image](https://github.com/vijaygill/set-bluetooth-clocks/assets/8999486/eec10f31-ddbd-4481-88df-0ca9381b73f4)

Is the bluetooth service running on host?
![image](https://github.com/vijaygill/set-bluetooth-clocks/assets/8999486/922847bc-2746-4621-931b-4d8052fb9973)

**Comment by Toasterme** [comment]
No, my bluetooth usb dongle is on my home assistant to be able to see the xiaomi device into it.  
Okay so i need another usb dongle to this machine then? thanks.

**Comment by vijaygill** [comment]
> No, my bluetooth usb dongle is on my home assistant to be able to see the xiaomi device into it. Okay so i need another usb dongle to this machine then? thanks.

I am using raspberry pi's (a 4b+) built-in bluetooth, so I have not tested it with a dongle.

But should you really need another dongle just for this? I will try to find more details about bluetooth dongles and sharing of those in various applications.

I have home assistant running on one raspberry pi (again a 4b+ and using its built-in bluethooth). I will run this container on it and share the findings with you.

**Comment by vijaygill** [comment]
@Toasterme -
I tried it on my raspberry pi where home-assistant is running and I see the container starts OK and also detects some bluetooth clocks.
![image](https://github.com/vijaygill/set-bluetooth-clocks/assets/8999486/7d5289ec-78c6-4dcc-8da3-b9075811951a)

Now, here the bluetooth is set as per Home-Assistant bluethooth integration. I have dbus-broker installed.
![image](https://github.com/vijaygill/set-bluetooth-clocks/assets/8999486/8029271c-c340-43d3-9070-92d8d6bb8797)

Also, I am not sure if it makes any difference, I have "--experimental" enabled for bluethooth service.

![image](https://github.com/vijaygill/set-bluetooth-clocks/assets/8999486/7adfc35a-8e3e-485f-89ac-b77fc06d1298)
